objax/module.py

```diff
@@ -74,6 +74,12 @@ def vars(self, scope: str = '') -> VarCollection:
                 vc.update(v.vars(scope=f'{scope}[{p}]'))
         return vc
 
+    def __getitem__(self, key: Union[int, slice]):
```

Comment:
Do we need docstring for RST documentation?

Reply:
I don't think so. It's the standard list square bracket operator overloaded so it doesn't return a list but instead a ModuleList or Sequential. It's not meant to show up in the documentation either.